### ISSUE TITLE
Laravel 7: Channels current state is unavailable

### ISSUE THREAD

**Issue by fylzero** [beyondcode/laravel-websockets#372]
When installing on Laravel 7 I can't seem to get the WebSockets dashboard to connect.

I've setup Laravel Websockets on several Laravel 5.8/6 apps and am fairly familiar with how to install it, but on Laravel 7 it doesn't seem to work.

1. I run php artisan websockets:serve
2. See that it says "Starting the WebSocket server on port 6001..."
3. I go to /laravel-websockets
4. Try to connect...
5. It just always says "Channels current state is unavailable".
6. Network tab says "WebSocket opening handshake timed out"

**Comment by hemant17** [comment]
@fylzero  i just install it in laravel 7 localhost and also in live server in both condition it's working fine

**Comment by fylzero** [comment]
@hemant17 Must be something on my end then.  I can't seem to find anyone with the same problem.

**Comment by dalholm** [comment]
> @hemant17 Must be something on my end then. I can't seem to find anyone with the same problem.

@fylzero i have the exact same problem. Did you find a solution ? :)

**Comment by fylzero** [comment]
@dalholm I wound up creating a fork of the laravel websockets demo project and updated it for Laravel 7 here.  It works, so it doesn't seem to have anything to do with the package being broken.  I gave up trying to pin point what I was doing wrong.

Here is my fork, which I've submitted as a PR to the laravel websockets demo project.
https://github.com/fylzero/laravel-websockets-demo/tree/update-to-laravel-7
